raiden-cli/src/routes/testing.ts

```diff
@@ -1,12 +1,30 @@
-import { Router, Request, Response } from 'express';
+import { Router, Request, Response, NextFunction } from 'express';
+import { ErrorCodes } from 'raiden-ts';
+import { validateAddressParameter } from '../utils/validation';
 import { Cli } from '../types';
 
+async function mintTokens(this: Cli, request: Request, response: Response, next: NextFunction) {
+  try {
+    response.json({
+      transaction_hash: await this.raiden.mint(request.params.tokenAddress, request.body.value, {
+        to: request.body.to,
+      }),
+    });
+  } catch (error) {
+    if ([ErrorCodes.DTA_INVALID_ADDRESS, ErrorCodes.DTA_INVALID_AMOUNT].includes(error.message))
+      response.status(400).send('Invalid "value" or "to" fields.');
```

Comment:
Usually we directly forward the error message here. So this if-else is basically in first place to determine the error code to use. Else we should avoid to attempt introducing new error messages and simply use the probably way better messages by the SDK.

Reply:
Done.